tests/smoke_tests/run_smoke_test.py

```diff
@@ -15,9 +15,17 @@
 
 
 async def run_smoke_test(
-    config_path: str = "tests/smoke_tests/config.yaml",
-    dataset_path: str = "examples/datasets/mnist_data/",
+    server_python_path: str,
+    client_python_path: str,
+    config_path: str,
+    dataset_path: str,
 ) -> None:
+    logger.info("Running smoke tests with parameters:")
```

Comment:
Maybe add some documentation to the method signature here in the Google Docstring style?